scripts/globals/keyitems.lua

```diff
@@ -289,7 +288,7 @@ xi.keyItem =
     RHINOSTERY_RING                          = 280,
     MANUSTERY_RING                           = 281,
     GLOVE_OF_PERPETUAL_TWILIGHT              = 282,
-    ANCIENT_SANDORIAN_TABLET                 = 283,
+    ANCIENT_SAN_DORIAN_TABLET                = 283,
```

Comment:
we strip out `'` not replace with underscore, so this needs to return to what it was